library/src/morpheus-master/sass/modules/_main.scss

```diff
@@ -69,6 +69,11 @@
 	transition:var(--sak-theme-switch-ease);
 }
 
+//Allow theme switching to work in iframes
```

Comment:
This isn't a holistic iframe solution. As discussed in the ui steering call last week, some work in portal will have to be done for them to properly work with dark theme.